app/views/users/_org_member.html.erb

```diff
@@ -7,7 +7,7 @@
   <% end %>
 </div>
 <h3>Danger zone</h3>
-<%= form_tag "/users/leave_org", onsubmit: "return confirm('Are you sure you want to leave this organization?');" do %>
+<%= form_tag "/users/leave_org/#{@organization.id}", onsubmit: "return confirm('Are you sure you want to leave this organization?');" do %>
```

Comment:
I suggest specifying a name for this route in `routes.rb` like `, as: :user_leave_org` and using a route helper `user_leave_org_path(@organization.id)` . This way there would be less chance that the required attribute will be missed like it was before the fix.

Reply:
Thanks for the suggestion. I've made the change and named the route as `users_leave_org` to be consistent with `users_join_org`!